packages/python/src/armonik/worker/worker.py

```diff
@@ -74,3 +79,87 @@ def Process(self, request: ProcessRequest, context) -> Union[ProcessReply, None]
 
     def HealthCheck(self, request: Empty, context) -> HealthCheckReply:
         return HealthCheckReply(status=self.health_check())
+
+
+def armonik_worker(
+    *,
+    autorun: bool = False,
+    logger: Optional[Logger] = None,
+    worker_endpoint: Optional[str] = None,
+    agent_endpoint: Optional[str] = None,
+    channel_options: Tuple[Tuple[str, str]] = (("grpc.default_authority", "localhost"),),
```

Comment:
I would argue this should not be arguments of the decorator, but arguments to the run function.
Also, it might be simpler to construct an object instead of nesting functions like that.